int32 count
gvgexploration/GoalPath[] current_paths
---
geometry_msgs/Pose[] frontiers
